docs/pages/connect-your-client/tsh.mdx

```diff
@@ -849,6 +849,33 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+## Debug logs
+
+Adding the `--debug` flag to a command or setting the `TELEPORT_DEBUG` env var to `1` makes tsh
+print debug logs to standard output.
+
+<Admonition type="note" >
+On macOS, the `--os-log` flag can be used instead of `--debug` to send debug logs to [the unified
+logging system](https://support.apple.com/en-gb/guide/console/welcome/mac). This behavior can also be controlled through the `TELEPORT_OS_LOG` env var.
```

Comment:
```suggestion
On macOS, the `--os-log` flag can be used to redirect application logs to [the unified
logging system](https://support.apple.com/en-gb/guide/console/welcome/mac). You can also enable this feature by setting the `TELEPORT_OS_LOG` env var to `1`. `--os-log` implies `--debug`.
```

Should we also mention Windows here?

Reply:
I didn't end up implementing Event Log integration for tsh on Windows, it is supported only in the VNet service. This is because it requires shipping a DLL and running install/uninstall steps as an admin. Both things are something we can feasibly do only in Connect at the moment.